index.html

```diff
@@ -181,6 +181,27 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var fireballSize = 22;
+  var wizardSpeed = 3;
+  var wizardWidth = 70;
+
+  function getFireballSpeed (isWindFromLeft) {
+    return isWindFromLeft?5:2;
```

Comment:
Вообще, принято пробелы ставить в таких местах
```suggestion
    return isWindFromLeft ? 5 : 2;
```

Reply:
Давай это тоже исправим)